app/serializers/person_escort_record_serializer.rb

```diff
@@ -1,34 +1,33 @@
-class PersonEscortRecordSerializer < ActiveModel::Serializer
-  belongs_to :profile, record_type: :profile
```

Comment:
Why the record_type, here? I can't seem to find anything in https://www.rubydoc.info/gems/active_model_serializers

Reply:
Grepping for it in the ams repo doesn't yield anything, either:

```
william  11:26  ~/R/active_model_serializers  > ag record_type
william  11:26  ~/R/active_model_serializers  >
````

Reply:
@cwrw any ideas? It did seem redundant to me so I removed it.